demo_scripts/simdem/syntax/README.md

```diff
@@ -51,6 +54,15 @@ Results:
 Hello world indent
 ```
 
+If a code block exists with a different language hint, it will not be
+execute, but instead will be output as formatted code, for example:
+
+```json
+{
+  Foo: "bar"
+}
+```
+
```

Comment:
Same issue as above happens here. When I remove the indented command, the program ends immediately after this command

Reply:
All 21 tests now pass.